docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -441,6 +441,12 @@ Property Name                                      Description
 
 ``location``                                       Optionally specifies the file system location URI for
                                                    the table.
+
+``format_version``                                 Optionally specifies the format version of the Iceberg
+                                                   specification to use for new tables; either ``1`` or ``2``.
+                                                   Defaults to ``1``. Version 2 is required for row level deletes.
+                                                   The full list of changes from v1 to v2 are outlined
+                                                   `here <https://iceberg.apache.org/spec/#version-2-row-level-deletes>`_.
```

Comment:
https://iceberg.apache.org/spec/#version-2 would be a correct link.
However, i am fine with omitting the whole sentence, we can consider this out of scope of this document.

From Trino user perspective, we should say that "2" is required for DML.

Reply:
> From Trino user perspective, we should say that "2" is required for DML

I was going to put that, but not until after DML is in

Reply:
Removed the last sentence